util/pace/util/grid/generation.py

```diff
@@ -2128,7 +2136,7 @@ def _compute_area_c_cartesian(self):
         area_cgrid_64.data[:, :] = self._dx_const * self._dy_const
         return quantity_cast_to_model_float(self.quantity_factory, area_cgrid_64)
 
-    def _set_hybrid_pressure_coefficients(self):
+    def _set_hybrid_pressure_coefficients(self, eta_file="None"):
```

Comment:
Yeah I think you should call this in the `__init__` method and save the outputs as `self._ak`, `self._bk`, `self._ptop`, and `self._ks`